rest/OrderService.js

```diff
@@ -2,35 +2,66 @@ import ProductPersistence from "../persistence/ProductPersistence.js";
 import OrderPersistence from "../persistence/OrderPersistence.js";
 import IllegalArgumentError from "./IllegalArgumentError.js";
 
+const EMPTY_ARGUMENT_ERR = "Body contains an empty argument!";
+const INVALID_TIMESTAMP_ERR = "Timestamp create should be now!";
```

Comment:
Timestamps will be generated by the server, they will not be part of the user input.

Also this error message is not grammatically correct + saying "now" to the client can mean many things depending on when they see the message.